src/test/java/alice/parser/ParserTest.java

```diff
@@ -0,0 +1,17 @@
+package alice.parser;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.time.LocalDateTime;
+
+import org.junit.jupiter.api.Test;
+
+import alice.command.InvalidCommandException;
+
+public class ParserTest {
+    @Test
+    void parseDateTimeTest() throws InvalidCommandException {
+        LocalDateTime dateTime = Parser.parseDateTime("12-Aug 1350");
+        assertEquals(LocalDateTime.of(2020, 8, 12, 13, 50), dateTime);
+    }
+}
```

Comment:
Given the multiple formats allow, perhaps you would want to test other valid and invalid inputs as well.

Reply:
The JUnit test is still an ongoing work and due to time constraint, I did not include a lot of tests. 

I will most likely add on this test in the future updates. Thanks for the input! 👍